src/web/lib/hakuneko/frontend@classic-dark/mangas.html

```diff
@@ -262,11 +295,41 @@
                 //
             }
 
+            /**
+            * Event listener attached to the refresh button
+            * Checks for new content to read by refreshing the bookmarks, updating the web source and checking the chapters bookmarks
+            */
+            async onCheckNewContent() {
+                if(! this.get('enableCheckNewContent'))
+                {
+                    alert('Please wait 60s before checking again.');
+                    return;
+                }
+                this.set('nbContentUpdating',Engine.BookmarkManager.bookmarks.length); 
+                await Engine.BookmarkManager.checkNewContent();
+                this.set('nbContentUpdating',0);
+                this.set('enableCheckNewContent',false);
+                //Disable refresh for 1 minute to prevent hammering
+                setTimeout(function(context){
+                    context.set('enableCheckNewContent',true);
+                },60000,this);
+            }
+
             /**
              *
              */
-            getMangaClass( selectedManga, id ) {
-                return ( !selectedManga || selectedManga.id !== id ? '' : 'focus' );
+            getMangaClass( selectedManga, manga ) {
```

Comment:
May reduce nested conditions to make code more understandable, e.g. classes array:
```javascript
let classes = [];
if(conditionSelected) {
    classes.push('classSelected')
}
if(conditionNewContent) {
    classes.push('classNewContent')
}
return class.join(' ');
```